lib/Common/Memory/HeapBlock.cpp

```diff
@@ -187,7 +187,7 @@ SmallHeapBlockT<TBlockAttributes>::ConstructorCommon(HeapBucket * bucket, ushort
     this->Init(objectSize, objectCount);
     Assert(heapBlockType < HeapBlock::HeapBlockType::SmallAllocBlockTypeCount + HeapBlock::HeapBlockType::MediumAllocBlockTypeCount);
     Assert(objectCount > 1 && objectCount == (this->GetPageCount() * AutoSystemInfo::PageSize) / objectSize);
-#ifdef RECYCLER_SLOW_CHECK_ENABLED
+#if defined(RECYCLER_SLOW_CHECK_ENABLED) || (ENABLE_CONCURRENT_GC && ENABLE_ALLOCATIONS_DURING_CONCURRENT_SWEEP)
     heapBucket->heapInfo->heapBlockCount[heapBlockType]++;
```

Comment:
Where is this used in release bits for your feature?

Reply:
Good catch. We don't use this but the count on the heap bucket. I will undo this change.

---
In reply to: [148563397](https://github.com/Microsoft/ChakraCore/pull/4104#discussion_r148563397) [](ancestors = 148563397)